Document `/meeting.schedule`

tiagobastos opened the pull request:
Based on ~https://github.com/teamleadercrm/api/pull/744~ https://github.com/teamleadercrm/api/pull/739

tiagobastos committed:
chore| add meeting.schedule

tiagobastos committed:
chore| fix bad indentation on assignees

tiagobastos committed:
Merge remote-tracking branch 'origin/ATL-2280-meeting-complete' into meetings.schedule

tiagobastos committed:
chore| fix attendees property & order they are displayed (for consistency)

tiagobastos committed:
chore| add location

Jdww commented on apiary.apib:
```diff
@@ -3316,6 +3316,63 @@ Get information about a meeting.
+        + starts_at: `2023-10-01T09:00:00+01:00` (string, required)
+        + ends_at: `2023-10-01T10:00:00+01:00` (string, required)
+        + description (string, optional)
+        + attendees (array, required) - at least one user assignee must be present
```
PP: at least one user attendee

tiagobastos committed:
fix| use correct wording

tiagobastos committed:
fix| /meetings.info location address

The renderer has an issue with `One of` and doesn't correctly
display data structures.

joaoinacio committed:
Merge pull request #746 from teamleadercrm/meetings.info
